### PR TITLE
Update clair3 and ability to provide correct Clair3 variant model

### PR DESCRIPTION
**Pull request by nhhaidee**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/CFIA-NCFAD/nf-flu/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on [the CFIA-NCFAD/nf-test-datasets repo](https://github.com/CFIA-NCFAD/nf-test-datasets/pull/new)
- [x] Make sure your code lints (`nf-core lint .`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by nhhaidee** [comment]
The test failed with Clair3 version 1.0.1

Please see issue https://github.com/HKU-BAL/Clair3/issues/198

**Comment by nhhaidee** [comment]
Clair3 version v0.1-r10 still works with `--clair3_user_variant_model`. Waiting for response for issue https://github.com/HKU-BAL/Clair3/issues/198.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,10 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [[3.1.4](https://github.com/CFIA-NCFAD/nf-flu/releases/tag/3.1.4)] - 2023-05-12
+
+Update Clair3 to version 1.0.1, option `--clair3_user_variant_model` allows to provide correct variant model which is not included with Clair3.
```
Comment (peterk87) [inline CHANGELOG.md:8]: Change to:

This release addresses #11 adding a new option, `--clair3_user_variant_model <PATH TO CLAIR3 MODEL>`, to allow user to provide a Clair3 model not included with Clair3, e.g. a [Rerio](https://github.com/nanoporetech/rerio) Clair3 model for r10 flowcells.

File: docs/usage.md [hunk 2]
```diff
@@ -183,7 +183,15 @@ Medaka model for initial SNP calling from mixed reads prior to phasing
...
-Set Clair3 Variant model
+Set Clair3 Available Variant model
+
+#### `--clair3_user_variant_model`
+
+- Optional
+- Type: string
+- Default: ''
+
+Set Clair3 User specific Variant model
```
Comment (peterk87) [inline docs/usage.md:194]: Change to:

Specify custom Clair3 model, e.g. [Rerio](https://github.com/nanoporetech/rerio) Clair3 model for R10 flowcells (`--clair3_user_variant_model /path/to/rerio-models/r1041_e82_400bps_sup_g615_model`)

**Review by peterk87 — CHANGES_REQUESTED** [review]
Hi @nhhaidee 

I've suggested a few changes. The version also needs to be updated in `nextflow.config` to 3.1.4.

File: docs/usage.md (hunk 2 above)
Comment (nhhaidee) [reply docs/usage.md]: Updated

**Review by peterk87 — APPROVED** [review]

**Comment by peterk87** [comment]
Thanks @nhhaidee 👍

**Pull request merged**